### PR TITLE
Update common files

### PR DESCRIPTION
**Pull request by klieret**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,71 +20,50 @@ agrees to abide by our [code of conduct](CODE_OF_CONDUCT.md).
 ## How to Contribute
 
 The easiest way to get started is to file an issue
-to tell us about a spelling mistake,
+to tell us about a mistake,
 some awkward wording,
-or a factual error.
+a factual error.
```
Comment (henryiii) [inline CONTRIBUTING.md:25]: This was better with the "or", IMO. :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -20,71 +20,50 @@ agrees to abide by our [code of conduct](CODE_OF_CONDUCT.md).
...
-    please work in <https://github.com/hsf-training/hsf-styles>.
-
 ## What to Contribute
 
 There are many ways to contribute,
 from writing new exercises and improving existing ones
 to updating or filling in the documentation
 and submitting [bug reports][issues]
-about things that don't work, aren't clear, or are missing.
+about things that do not work, aren not clear, or are missing.
```
Comment (henryiii) [inline CONTRIBUTING.md:52]: Aren not?

File: CONTRIBUTING.md (hunk 1 above)
Comment (klieret) [reply CONTRIBUTING.md:25]: indeed

File: CONTRIBUTING.md (hunk 2 above)
Comment (klieret) [reply CONTRIBUTING.md:52]: thanks

**Comment by klieret** [comment]
I'll merge this now. I've fixed the files in the maintenance repository, so your comments will be fixed when I sync again in the next few days (because it affects all repos).

**Pull request merged**